### PR TITLE
Fix #5324 UI: Add ML model entity on explore page.

### LINKED ISSUE #5324: UI: Add ML model entity on explore page.
**Feature**
Add feature issue reference

**Describe the task**

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Fixes #5324 
### Describe your changes :

I worked on explore page for adding the ml model entity.
#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Frontend Preview (Screenshots) :
image

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

**Comment by vivekratnavel** [comment]
@Sachin-chaurasiya This looks good. Can we say "ML Models" instead of "MlModels" in the search box as well as the tab?

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya This looks good. Can we say "ML Models" instead of "MlModels" in the search box as well as the tab?

Done, Please see the updated screenshots.

File: openmetadata-ui/src/main/resources/ui/src/components/app-bar/Suggestions.tsx [hunk 1]
```diff
@@ -116,6 +133,12 @@ const Suggestions = ({ searchText, isOpen, setIsOpen }: SuggestionProp) => {
         label = 'Pipelines';
         icon = Icons.PIPELINE_GREY;
 
+        break;
+      case SearchIndex.MLMODEL:
+        label = 'MlModels';
```
Comment (vivekratnavel) [inline openmetadata-ui/src/main/resources/ui/src/components/app-bar/Suggestions.tsx:138]:
```suggestion
        label = 'ML Models';
```

File: openmetadata-ui/src/main/resources/ui/src/constants/explore.constants.ts [hunk 2]
```diff
@@ -260,4 +268,14 @@ export const tabsInfo = [
     icon: Icons.PIPELINE_GREY,
     selectedIcon: Icons.PIPELINE,
   },
+  {
+    label: 'MlModels',
```
Comment (vivekratnavel) [inline openmetadata-ui/src/main/resources/ui/src/constants/explore.constants.ts:272]:
```suggestion
    label: 'ML Models',
```

File: openmetadata-ui/src/main/resources/ui/src/axiosAPIs/miscAPI.ts [hunk 3]
```diff
@@ -75,7 +75,7 @@ export const getSuggestions: Function = (
   return APIClient.get(
     `/search/suggest?q=${queryString}&index=${
       searchIndex ??
-      `${SearchIndex.DASHBOARD},${SearchIndex.TABLE},${SearchIndex.TOPIC},${SearchIndex.PIPELINE}`
+      `${SearchIndex.DASHBOARD},${SearchIndex.TABLE},${SearchIndex.TOPIC},${SearchIndex.PIPELINE},${SearchIndex.MLMODEL}`
```
Comment (yug-shah0106) [inline openmetadata-ui/src/main/resources/ui/src/axiosAPIs/miscAPI.ts:78]: can we please use queryParams as an object and not append it manually ?

File: openmetadata-ui/src/main/resources/ui/src/components/app-bar/Suggestions.tsx [hunk 4]
```diff
@@ -244,6 +269,24 @@ const Suggestions = ({ searchText, isOpen, setIsOpen }: SuggestionProp) => {
...
           </>
         )}
+        {mlModelSuggestions.length > 0 && (
+          <>
+            {getGroupLabel(SearchIndex.MLMODEL)}
+
+            {mlModelSuggestions.map((suggestion: MlModelSource) => {
+              const fqdn = suggestion.fqdn;
+              const name = suggestion.name;
+              const serviceType = suggestion.service_type;
```
Comment (yug-shah0106) [inline openmetadata-ui/src/main/resources/ui/src/components/app-bar/Suggestions.tsx:279]: NIT : const {fqdn,name, service_type:serviceType} = suggestion;

File: openmetadata-ui/src/main/resources/ui/src/axiosAPIs/miscAPI.ts (hunk 3 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/axiosAPIs/miscAPI.ts:78]: @yug-shah0106 , Yes, will update this.

**Pull request merged**